### PR TITLE
directory creation error resolved

### PR DESCRIPTION
**Pull request by venkateshchary**

### CODE REVIEW

File: src/plogutils.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def __init__(self):
 
         # Config Varables
         self._to_file = False
-        self._file_location = '/var/log/plogs/'
+        self._file_location = '{0}'.format('{0}'.format(os.getcwd()+"\\logs\\"))
```
Comment (venkateshchary) [inline src/plogutils.py]: {0}.format(os.getcwd()+"\\logs\\")) 
format is copied 2 times use above line

File: src/plogutils.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def table(self, objects):
         table_str = construct_table(objects)
         self._log(table_str, Levels.STATUS)
 
-    def config(self, pretty=True, show_levels=False, show_time=False, to_file=False, file_location='/var/log/plogs/', filename='plogs_01.log'):
+    def config(self, pretty=True, show_levels=False, show_time=False, to_file=False, file_location='{0}'.format('{0}'.format(os.getcwd()+"\\logs\\")), filename='plogs_01.log'):
```
Comment (venkateshchary) [inline src/plogutils.py]: '{0}'.format(os.getcwd()+"\logs\")
format is copied 2 times use above line

**Comment by 11** [comment]
Wow thank you for forking this and looking into the issue A lot of these problems are actually being fixed in the next release which is why I haven't pushed out a working version. Def want to give this a quick test on both windows, linux, and OSX. 

But if all works, I will merge and as a temp merge while the next major release is being worked on

**Comment by venkateshchary** [comment]
> Wow thank you for forking this and looking into the issue A lot of these problems are actually being fixed in the next release which is why I haven't pushed out a working version. Def want to give this a quick test on both windows, linux, and OSX.
> 
> But if all works, I will merge and as a temp merge while the next major release is being worked on

ok i will test it on , can i contribute to your major changes.

**Pull request closed without merging**